x-pack/plugins/enterprise_search/common/types/engines.ts

```diff
@@ -14,15 +14,9 @@ export interface EnterpriseSearchEnginesResponse {
 }
 
 export interface EnterpriseSearchEngine {
-  indices: string[];
-  name: string;
-  updated_at_millis: number;
-}
-
-export interface EnterpriseSearchEngineDetails {
```

Comment:
We'll still need this type to be different from `EnterpriseSearchEngine` since the list and get responses return different data. List results has `indices: string[]` and should use `EnterpriseSearchEngine` and the get for a single search application will need to return a `EnterpriseSearchEngineDetails` which has the indices hydrated.